controllers/wizard/fileUpload/FileUploadWizardHandler.php

```diff
@@ -415,7 +415,7 @@ public function uploadFile($args, $request)
 
         // Validate the form and upload the file.
         if (!$uploadForm->validate()) {
-            return new JSONMessage(false, $uploadForm->fetch($request));
```

Comment:
The pattern of including the form markup in the JSON failure message was to replace the "failed" form markup with an updated version that might include more details. If it helps: this is processed e.g. in the AjaxFormHandler in our old JS stack; see e.g.:

https://github.com/pkp/pkp-lib/blob/bf99547b8cbe7dd3014b306c165fabb6b05372f8/js/controllers/form/AjaxFormHandler.js#L123..L140